src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx

```diff
@@ -0,0 +1,143 @@
+
+Similar to the map view, you can click on specific entities (services) to see transactions in the drill-down tables. As you drill down and find an interesting endpoint, you can click on **Traces** to switch over to distributed tracing details.
+
+To the right of each performance metric column is a corresponding **% change** (percentage change) column. The percentage change calculation is based on the time frame you choose in the main time picker and the comparison time picker (**Compare to**). The comparison time picker indicates how much before the main time window the comparison should start. 
+
+For example, with **Response time**, if the current time is 11 a.m. and the main time picker is **last 30 minutes** and the **Compare to** time picker is **1 hour ago**, the duration is the average from 10:30-11:00 a.m., and the **% change** compares that to the average from 9:30-10:00 a.m.
+
+## Workflow example [#workflow-example]
+
+Here is a typical map workflow you might follow:
+
+1. Look for the thickest and darkest line on the map and follow it to its upstream or downstream service.
+2. Click on the upstream or downstream vertex.
+3. View a breakdown of transactions between the two services.
+![Screenshot showing calls to various endpoints in downstream service](./images/downstream-endpoints.png "Screenshot showing calls to various endpoints in downstream service")
+<figcaption>In this example, one of the thicker edges (lines) goes from the Order-Composer service to the warehouse endpoint in the Order Status service.</figcaption>
+4. If you decide that a particular transaction is taking the most time, you can click on that transaction to focus specifically on its dependencies.
+![Screenshot showing the transaction between the Order-Composer service and the warehouse endpoint in the Order-Status service](./images/specific-transaction.png "Screenshot showing the transaction between the Order-Composer service and the warehouse endpoint in the Order-Status service")
+<figcaption>In this drill-down view, you can see the transaction between the Order-Composer service and the warehouse endpoint in the Order-Status service.</figcaption>
+5. From any point in this flow, consult the supporting performance metrics, which show changes over time.
+6. If you reach a point in the drill-down where you want to see distributed tracing, click **List** in the upper-right, and then click **Traces** in the table.
+![Screenshot showing the trace link in the table view](./images/tracing-link-in-table.png "Screenshot showing the trace link in the table view")  
+
+## How to view classic external services [#classic-toggle]
+
+The classic external services view is still available if you are monitoring existing services that use cross application tracing. Since the default view is the expanded external services, you need to click the toggle **Show new view** to switch to the classic view. 
```

Comment:
no dash for CAT this time, woo hoo! :-)

Reply:
@barbnewrelic, sometimes I get it right!! Ha!